src/Spec2-Commander2/SpCommand.class.st

```diff
@@ -121,7 +131,13 @@ SpCommand >> iconProvider: anObject [
 SpCommand >> initialize [ 
 
 	super initialize.
-	self beDisabledWhenCantBeRun
+	self beDisabledWhenCantBeRun.
```

Comment:
should be `self beDisableWhenCannotBeExecuted`

Reply:
Yes, but for another issue